packages/drawer/src/types.tsx

```diff
@@ -220,6 +220,10 @@ export type DrawerNavigationOptions = HeaderOptions & {
    * Only supported on iOS and Android.
    */
   freezeOnBlur?: boolean;
+  /**
+   * If set, screen readers will read out this string when the user presses on the overlay adjacent to an opened drawer, which will close the drawer.
+   * **/
+  closeAccessibilityLabel?: string;
```

Comment:
Can you change it to `overlayAccessibilityLabel`? Also, move it next to `overlayColor` so that related props are grouped

Reply:
Can do! Will add to next push.